inspirehep/config.py

```diff
@@ -886,7 +886,7 @@ def _(x):
 
 # Web services and APIs
 # =====================
-BEARD_API_URL = None  # e.g. "http://beard.inspirehep.net/api"
+BEARD_API_URL = "http://beard.inspirehep.net/api"  # For testing purposes.
```

Comment:
Nope, that's a leftover. It was uncommented to get `phonetic_blocks` on the records, but I think there is another way now.